cli.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+const pentf = require('./index.js');
+
+const cwd = process.cwd();
+pentf.main({
+    description: 'pentf - Parallel End-To-End Test Framework',
+    rootDir: cwd,
+    testsDir: cwd,
```

Comment:
This doesn't look right – shouldn't the `rootDir` normally be on top of the tests dir?

Suggestion: Can we auto-detect?

Current directory contains `./config` or `../config` -> that's our configDir.
Current directory contains `./tests` -> that's our testDir.
Otherwise, testDir is `.` (aka cwd).

Reply:
I'd love for us to move away from making assumptions where things are. This can become problematic for projects where the `tests` folder is already used for unit tests. The `config` directory may also be already used by other tools. Not sure what the best way forward for that is though.

Reply:
Then how do we envision a layout with pentf and other tests? How about first looking for `pentf_config` and `pentf_tests`.

Reply:
For most frontend projects there never will be a single tests directory to look for. Instead tests are co-located with the feature/module that is developed.

```txt
/src
  /feature-a
    a.js
    a.test.js
  /feature-b
    b.js
    b.test.js
```

Another common pattern is:

```txt
/src
  /feature-a
    /__tests__
      a.test.js
    a.js
  /feature-b
    /__tests__
      b.test.js
    b.js
```

This scenario cannot be specified by single directory as it's not specific enough. In fact it was the main reason for adding support for globs in #204 . With globs merged we support all use cases, whatever the user prefers.

- `tests` directory -> `tests/*.js`
- co-located `__tests__` directory -> `**/__tests__/*.js`
- etc

Pretty much every frontend developer is already familiar with globs, because every frontend unit testing framework uses them.

When showing pentf to new users and letting them set it up without any guidance, they all got stuck by tests not being found. They attempted to resolve them via the glob argument, but teh `testsDir` was messing with that by changing the `cwd` from which globs are resolved from. So to remove that barrier/friction point I'm very much in support of removing `testsDir` alltogether and pointing users to just use globs. We can still keep supporting it to not introduce a breaking change, but we should default `testsDir` to an empty string.

Reply:
For reference: This is the default glob that is used by Jest
```txt
"**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)"
```